components/board/interrupt_streams_test.go

```diff
@@ -0,0 +1,24 @@
+package board
+
+import (
+	"testing"
+
+	"go.viam.com/test"
+)
+
+func TestRemoveStream(t *testing.T) {
+	c := &client{}
+
+	stream1 := &interruptStream{
+		client: c,
+	}
+	stream2 := &interruptStream{
+		client: c,
+	}
+
+	testStreams := []*interruptStream{stream1, stream2}
+	c.interruptStreams = testStreams
+	expectedStreams := []*interruptStream{stream1}
+	c.removeStream(stream2)
+	test.That(t, c.interruptStreams, test.ShouldResemble, expectedStreams)
```

Comment:
```suggestion
	stream2 := &interruptStream{
		client: c,
	}
	
	stream3 := &interruptStream{
		client: c,
	}

	testStreams := []*interruptStream{stream1, stream2, stream3}
	c.interruptStreams = testStreams
	expectedStreams := []*interruptStream{stream1, stream3}
	c.removeStream(stream2)
	test.That(t, c.interruptStreams, test.ShouldResemble, expectedStreams)
```

Reply:
This isn't in the pr right now - make sure it's added and passes unless I'm misunderstanding the flow.

Reply:
no my mistake - looking at an old commit.